README.md

```diff
@@ -1,67 +1,232 @@
 - Export of IDM configuration
 - Configuring a ForgeRock cloud tenant for a specific use case, also called a "recipe"
 
-## Future
+### Future
+
 - Importing IDM configuration
 - Parameter substitution when exporting and importing IDM configuration
 - Export and import of other ForgeRock ID Cloud configurations, like OAuth clients, SAML entities etc.
 - Export and import of data in ForgeRock ID Cloud
 
 ## Limitations
+
 `frodo` can't export passwords (including API secrets, etc), so these need to be manually added back to an imported tree or alternatively, export the source tree to a file, edit the file to add the missing fields before importing. Any dependencies _other than_ scripts and email templates, needed for a journey/tree, must also exist prior to import, for example inner-trees and custom nodes.
 
-# Quick start
+## Usage
+
 Depending on your OS, you can invoke `frodo-linux`, `frodo-macos` or `frodo-windows.exe` binary.
-`frodo` uses commands and sub-commands to provide the functionality stated above.
+`frodo` uses commands and sub-commands to provide the functionality stated above. All commands support the help argument as such
+
+```shell
+frodo journey -h
+```
+
+```shell
+Usage: frodo journey [options] [command]
+
+Manage journeys/trees.
+  ```
+
+### Commands
+
+- [connections](#connections)
+- [idm](#idm)
+- [info](#info)
+- [journey](#journey)
+- [logging](#logging)
 
-## Examples
-### Export a journey
+#### Connections
+
+Use this command to list, add, or delete your connections. The CLI creates connection configurations automatically when you use other commands, so you don't have to run this command to initiate a connection.
+
+##### Examples
+
+List saved connections
+
+```shell
+frodo connections list
 ```
-$ ./frodo-linux journey -h https://<forgerock-cloud-tenant>/am -u <tenant.admin@tenant.com> -p my_5up3rs3cr3tp@ssw0rd export -t Registration -r alpha -f Registration.json
+
+##### Sample output
+
+The command displays the connections found in the **`frodorc`** file, located in the user's directory.
+
+```shell
+[Host] : [Username]
+- [https://openam-ali-demo.forgeblocks.com/am] : [ali.falahi@forgerock.com]
+Any unique substring of a saved host can be used as the value for host parameter in all commands
 ```
-This will export a journey named `Registration` in `alpha` realm and save the contents in a file called `journey-Registration.json`. If the `-f <filename>` is omitted, the journey JSON data is dumped to STDOUT.
 
-### Import a journey
+The command below lists all journeys in the specified realm, but the main idea here is when you supply the host, username, and `password` frodo will automatically save the connection in the **`frodorc`** file. The `password` value will be encrypted with `AES-256`
```

Comment:
.frodorc, right? Did you move the .frodorc file into the .frodo folder as well? Or is that still in ~?

Reply:
Moved. Will update the reader accordingly